src/utils/getVerifyProvidersStep.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { VerifyProvidersStep } from "@microsoft/vscode-azext-azureutils";
+import type { ISubscriptionActionContext } from "@microsoft/vscode-azext-utils";
+import { appProvider, operationalInsightsProvider, registryProvider, webProvider } from "../constants";
+
+/**
+ * Use to obtain a `VerifyProvidersStep` that registers all known container app providers to the user's subscription
+ */
+export function getVerifyProvidersStep<T extends ISubscriptionActionContext>(): VerifyProvidersStep<T> {
```

Comment:
I don't think you need to technically provide a context here based off the old way we used to create them.

Reply:
Yeah that makes sense, I'll just have to return using type inference instead